app/src/main/res/values/strings_settings.xml

```diff
@@ -66,8 +66,8 @@
     <string name="windowbar_mode_nav">Hide navigation bar</string>
     <string name="windowbar_mode_both">Hide both</string>
 
-    <string name="gestures_left">Swipe left</string>
-    <string name="gestures_right">Swipe right</string>
+    <string name="gestures_left">Swipe Left</string>
```

Comment:
I much prefer ← and →, and "up" is not consistent if this is "Left". I don't think it qualifies as a headline.